### PR TITLE
[OR] Update links for China help

### PR DESCRIPTION
**Pull request by Hixie**
We have a page on the Web page now.

### CODE REVIEW

File: bin/internal/update_dart_sdk.sh [hunk 1]
```diff
@@ -55,18 +55,18 @@ if [ ! -f "$ENGINE_STAMP" ] || [ "$ENGINE_VERSION" != `cat "$ENGINE_STAMP"` ]; t
 
   curl --continue-at - --location --output "$DART_SDK_ZIP" "$DART_SDK_URL" 2>&1 || {
     echo
-    echo "Failed to retrieve the Dart SDK at $DART_SDK_URL"
-    echo "If you're located in China, please follow"
-    echo "https://github.com/flutter/flutter/wiki/Using-Flutter-in-China"
+    echo "Failed to retrieve the Dart SDK from: $DART_SDK_URL"
```
Comment (goderbauer) [inline bin/internal/update_dart_sdk.sh]: The Windows version of this file `update_dart_sdk.ps1` should probably also include this warning. Some users in China may be on Windows.

**Review by goderbauer — APPROVED** [review]
LGTM

File: bin/internal/update_dart_sdk.sh (hunk 1 above)
Comment (Hixie) [reply bin/internal/update_dart_sdk.sh]: Looks like that script doesn't know how to determine if the download failed. I'll file a bug about adding that; I would rather my "change some strings" PR didn't turn into a "learn powershell" PR... :-)

Comment (goderbauer) [reply bin/internal/update_dart_sdk.sh]: Somebody once told me "embrace the yak shave!" 😛

Comment (Hixie) [reply bin/internal/update_dart_sdk.sh]: I'm knee-deep in yak hair as it is. :-P

**Pull request merged**